frontend/src/features/admin-form/settings/components/SecretKeyActivationModal.tsx

```diff
@@ -213,6 +228,7 @@ export const SecretKeyActivationModal = ({
                 rightIcon={<BiRightArrowAlt fontSize="1.5rem" />}
                 type="submit"
                 isFullWidth
+                hidden={activateHidden}
```

Comment:
should we change the "Activate form" button to be disabled instead of hidden?

```suggestion
                hidden={!watchedSecretKey}
                isDisabled={!watchedAck}
```